integration/data/utils.py

```diff
@@ -38,20 +38,20 @@ def create_backup(backup_target, snap):
     return backup
 
 
-def rm_backups(*backups):
+def rm_backups(url, *backups):
```

Comment:
We can change *backups to an array and put `url` after it. It looks weird to have the empty inserted into every function.